mesonbuild/dependencies/base.py

```diff
@@ -910,20 +911,21 @@ def _set_libs(self):
         # paths so we can do manual searching with cc.find_library() later.
         env = os.environ.copy()
         env['PKG_CONFIG_ALLOW_SYSTEM_LIBS'] = '1'
-        ret, out = self._call_pkgbin(libcmd, env=env)
+        ret, out, err = self._call_pkgbin(libcmd, env=env)
         if ret != 0:
-            raise DependencyException('Could not generate libs for %s:\n\n%s' %
-                                      (self.name, out))
+            raise DependencyException('Could not generate libs for %s:\n%s\n' %
+                                      (self.name, err))
         # Also get the 'raw' output without -Lfoo system paths for adding -L
         # args with -lfoo when a library can't be found, and also in
         # gnome.generate_gir + gnome.gtkdoc which need -L -l arguments.
-        ret, out_raw = self._call_pkgbin(libcmd)
+        ret, out_raw, err_raw = self._call_pkgbin(libcmd)
         if ret != 0:
-            raise DependencyException('Could not generate libs for %s:\n\n%s' %
-                                      (self.name, out_raw))
-        ret, out_all = self._call_pkgbin(libcmd_all)
+            raise DependencyException('Could not generate libs for %s:\n%s\n' %
+                                      (self.name, err_raw))
+        ret, out_all, err_all = self._call_pkgbin(libcmd_all)
         if ret != 0:
-            mlog.warning('Could not determine complete list of dependencies for %s' % self.name)
+            mlog.warning('Could not determine complete list of dependencies for %s:\n%s\n' %
+                        (self.name, err_all))
```

Comment:
__[Flake8]__ continuation line under-indented for visual indent ([see on Sider](https://sider.review/gh/repos/19784232/pulls/5656#issue-4804515))

<sub>posted by Sider</sub>